scripts/setupvars/setupvars.sh

```diff
@@ -119,15 +119,20 @@ fi
 
 
 if [ -n "$python_version" ]; then
-    # add path to OpenCV API for Python 3.x
-    export PYTHONPATH="$INTEL_OPENVINO_DIR/python/python3:$PYTHONPATH"
-    pydir=$INTEL_OPENVINO_DIR/python/python$python_version
-    if [[ -d $pydir ]]; then
-        # add path to Inference Engine Python API
-        export PYTHONPATH="${pydir}:${PYTHONPATH}"
+    if [[ -d $INTEL_OPENVINO_DIR/python ]]; then
+        # add path to OpenCV API for Python 3.x
+        export PYTHONPATH="$INTEL_OPENVINO_DIR/python/python3:$PYTHONPATH"
+        pydir=$INTEL_OPENVINO_DIR/python/python$python_version
+        if [[ -d $pydir ]]; then
+            # add path to Inference Engine Python API
+            export PYTHONPATH="${pydir}:${PYTHONPATH}"
+        else
+            echo "[setupvars.sh] WARNING: Can not find OpenVINO Python module for python${python_version} by path ${pydir}"
+            echo "[setupvars.sh] WARNING: OpenVINO Python environment does not set properly"
+        fi
```

Comment:
Shall we do the same for Windows?

Reply:
Ok, we will discuss about requirements for Win, because Python3.6 - 3.8(64-bit) mentioned as requirement for OpenVINO.

Reply:
But it's not a requirement for [Deployment Manager](https://docs.openvinotoolkit.org/latest/openvino_docs_install_guides_deployment_manager_tool.html) output package (i.e. CPU plugin only without Python bindings).